_posts/0000-01-02-azaimirul.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
+Use the left arrow to go back!
```

Comment:
I really don't know what to do right now. The world is too corrupt and vile to me. I'm really need someone who can be on my side when I have a trouble besides know how to provide me with the good advice tally with my generations.